.github/workflows/site_encrypt.yaml

```diff
@@ -14,8 +14,8 @@ jobs:
 
       - uses: erlef/setup-elixir@v1
         with:
-          otp-version: 26.0
-          elixir-version: 1.16.0
+          otp-version: 27.0
+          elixir-version: 1.17.0
```

Comment:
1.17.1 has been released recently, should I update it? Or should we pin to 1.17 instead?